docs/servo_power_test/src/power_test_controller.ts

```diff
@@ -53,17 +57,36 @@ export class PowerTestController {
       console.log(`start running config${i}`);
       await this.configList[i].start();
     }
+    this.drawTotalHistogram();
   }
   public async stopMeasurement() {
     await this.configList[this.currentConfigNum].stop();
   }
+  private drawTotalHistogram() {
+    const histogramData = [];
+    for (const config of this.configList) {
+      const annotations = config.annotationList;
+      const extractedData = [];
+      for (const powerData of config.powerDataList) {
+        if (
+          annotations.get('start')! + this.MARGIN_TIME <= powerData[0] &&
+          powerData[0] <= annotations.get('end')! - this.MARGIN_TIME
+        ) {
+          extractedData.push(powerData[1]);
+        }
+      }
+      histogramData.push(extractedData);
+    }
+    this.totalHistogram.paintHistogram(histogramData);
```

Comment:
histogramData already doesn't have timestamp data. In the current implementations, it's 2D because it has power data for each config. That is, `[[config1 power data], [config2 power data]]`

Reply:
aaaaah, you're right. then my comment is not applicable. Thanks for the explanation!